From 241c414445c1447a4f2d29670ac77f1e75e95322 Mon Sep 17 00:00:00 2001
From: Rajat Srivastava <rajat.srivastava@nxp.com>
Date: Thu, 1 Feb 2018 16:35:34 +0530
Subject: [PATCH 029/107] ls2088a: qspi: Enable config to write aligned data to TxFIFO

Enable config in LS2088A boards to send only 16 bytes aligned
data to TxFIFO while writing to flash.

Signed-off-by: Rajat Srivastava <rajat.srivastava@nxp.com>
---
 configs/ls2088ardb_qspi_SECURE_BOOT_defconfig |    1 +
 configs/ls2088ardb_qspi_defconfig             |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configs/ls2088ardb_qspi_SECURE_BOOT_defconfig b/configs/ls2088ardb_qspi_SECURE_BOOT_defconfig
index b405a8e..94372e9 100644
--- a/configs/ls2088ardb_qspi_SECURE_BOOT_defconfig
+++ b/configs/ls2088ardb_qspi_SECURE_BOOT_defconfig
@@ -36,6 +36,7 @@ CONFIG_SYS_NS16550=y
 CONFIG_DM_SPI=y
 CONFIG_FSL_DSPI=y
 CONFIG_FSL_QSPI=y
+CONFIG_FSL_SPI_ALIGNED_TXFIFO=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_XHCI_HCD=y
diff --git a/configs/ls2088ardb_qspi_defconfig b/configs/ls2088ardb_qspi_defconfig
index 4ea8a5c..62b35c0 100644
--- a/configs/ls2088ardb_qspi_defconfig
+++ b/configs/ls2088ardb_qspi_defconfig
@@ -40,6 +40,7 @@ CONFIG_SYS_NS16550=y
 CONFIG_DM_SPI=y
 CONFIG_FSL_DSPI=y
 CONFIG_FSL_QSPI=y
+CONFIG_FSL_SPI_ALIGNED_TXFIFO=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_XHCI_HCD=y
-- 
1.7.1

